reactor-core/src/main/java/reactor/core/publisher/FluxPublishMulticast.java

```diff
@@ -169,6 +178,9 @@ public Object scanUnsafe(Attr key) {
 			if (key == Attr.BUFFERED) {
 				return queue != null ? queue.size() : 0;
 			}
+			if (key == RUN_STYLE) {
```

Comment:
`PublishMulticastInner` should be also covered

Reply:
Fixed in commit [26552f6](https://github.com/reactor/reactor-core/pull/2123/commits/26552f67f2cfd7c10e59e60f3b905504c55b6e1d)